waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -323,3 +324,10 @@ getPackageJsonOverrides = map buildOverrideData (designateLastElement overrides)
     designateLastElement l =
       map (\(x1, x2, x3) -> (x1, x2, x3, False)) (init l)
         ++ map (\(x1, x2, x3) -> (x1, x2, x3, True)) [last l]
+
+genEnvValidationScript :: Generator [FileDraft]
+genEnvValidationScript =
+  return
+    [ C.mkTmplFd [relfile|scripts/validate-env.mjs|],
+      C.mkSharedTmplFdWithDst [relfile|validators.js|] [relfile|scripts/shared/validators.mjs|]
```

Comment:
Really minor, but why not change the the template's extension to `mjs` as well?

Reply:
It's slightly related to the idea of `scripts` and `src` being two different scopes. For some other use case in` src` where we would use the validators, the extension won't be `mjs` but rather `js`. 

So, it's the renaming of the file that actually makes this file work in `scripts` scope. It's kinda implicit, if you have a better way of doing it, I'll hapiliy consider it.

Reply:
It's fine as far as I'm concerned.